pallets/phala/src/phala_legacy/tests.rs

```diff
@@ -163,19 +160,16 @@ fn test_register_worker() {
 		));
 		assert_ok!(Phala::set_stash(Origin::signed(1), 1));
 		// TODO(jasl): fix runtime info
-		assert_ok!(Phala::register_worker(
+		// TODO: we need to re-design the tests.
```

Comment:
We will will retire phala_legacy. So we can feel free to delete the out-of-date tests

Reply:
So I removed the whole test file. If we need it as a referance when we write new tests, we can recover it from git.